packages/fxa-payments-server/src/components/SubscriptionTitle/index.tsx

```diff
@@ -3,36 +3,45 @@ import { Localized } from '@fluent/react';
 
 import './index.scss';
 
-type SubscriptionTitleProps = {
-  screenType: 'create' | 'success' | 'processing' | 'error';
+export const titles = {
+  create: 'Set up your subscription',
+  success: 'Subscription confirmation',
+  processing: 'Confirming subscription...',
+  error: 'Error confirming subscription...',
+  noupgrade: 'Subscription tier changes are not supported',
+} as const;
+
+export type SubscriptionTitleProps = {
+  screenType: keyof typeof titles;
+  subtitle?: React.ReactElement;
   className?: string;
 };
 
 export const SubscriptionTitle = ({
   screenType,
+  subtitle,
   className = '',
 }: SubscriptionTitleProps) => {
-  const defaultHeaders: {
-    [K in SubscriptionTitleProps['screenType']]: string;
-  } = {
-    create: 'Set up your subscription',
-    success: 'Subscription confirmation',
-    processing: 'Confirming subscription...',
-    error: 'Error confirming subscription...',
-  };
+  const subtitleElement = subtitle ? (
+    subtitle
+  ) : (
+    <Localized id="sub-guarantee">
+      <p className="subtitle">30-day money-back guarantee</p>
+    </Localized>
+  );
```

Comment:
I did that at first, and it didn't work!  I was surprised.  But since it's syntax sugar provided by TypeScript, I didn't want to invest the time to dig into it.